src/clients/client.ts

```diff
@@ -184,6 +185,19 @@ class Client {
     }
   }
 
+  /**
+   * Swap two indexes
```

Comment:
The request supports more than two indexes, right?

Also, can you improve a little bit this description?